2022-02-28 11:27:05,006 [main] INFO com.classes.BaseTest - Loading all methods from -masterVO class
2022-02-28 11:27:05,008 [main] INFO com.utils._masterVO - Loading Properties file from: .//src//test//resources//sysconfig//Constant.properties
2022-02-28 11:27:05,012 [main] INFO com.utils._masterVO - Properties File Loaded successfully
2022-02-28 11:27:05,013 [main] INFO com.utils._masterVO - Loading Master Sheet from: src//test//resources//config//DataProvider.xlsx
2022-02-28 11:27:05,584 [main] INFO com.utils._masterVO - Master Sheet Loaded successfully
2022-02-28 11:27:05,586 [main] INFO com.utils._masterVO - Loading Client Library for Client Versioning from: .//src//test//resources//sysconfig//ClientLib.properties
2022-02-28 11:27:05,598 [main] INFO com.utils._masterVO - Client Library Loaded successfully
2022-02-28 11:27:05,841 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::ZO - AUTbKlI - AUToKa1C
2022-02-28 11:27:05,950 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::AR - AUT7dba - AUT63dCN
2022-02-28 11:27:06,041 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::SA - AUTO2Fm - AUTNelnh
2022-02-28 11:27:06,380 [main] INFO com.utils._masterVO - Loading test case file file from: .//src//test//resources//sysconfig//testCaseFile.properties
2022-02-28 11:27:06,389 [main] INFO com.utils._masterVO - Test cases file Loaded successfully
2022-02-28 11:27:06,391 [main] INFO com.utils._masterVO - Loading Web Cases Master file from: .//src//test//resources//config//testCaseFile.properties
2022-02-28 11:27:06,403 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-02-28 11:27:06,426 [main] INFO com.dbrepository.DBHandler - Entered Access Handler :: PostGreSQL
2022-02-28 11:27:06,430 [main] INFO com.commons.CacheController - New Cache DAO Loaded :: UPDATECACHE_VER = 1
2022-02-28 11:27:06,456 [MLog-Init-Reporter] INFO com.mchange.v2.log.MLog - MLog clients using log4j logging.
2022-02-28 11:27:06,534 [main] INFO com.mchange.v2.c3p0.C3P0Registry - Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
2022-02-28 11:27:06,619 [main] INFO com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 5, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge9geanhgbuwt152jeeh|4d6025c5, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> org.postgresql.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge9geanhgbuwt152jeeh|4d6025c5, idleConnectionTestPeriod -> 0, initialPoolSize -> 10, jdbcUrl -> jdbc:postgresql://172.30.38.120:5432/pretups_trunk, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2022-02-28 11:27:07,438 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getNetworkName()
2022-02-28 11:27:07,576 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: Network Name (Network GGN)
2022-02-28 11:27:07,578 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(AMOUNT_MULT_FACTOR)
2022-02-28 11:27:07,579 [main] INFO com.dbrepository.DBConnectionPool - Active Size:1 Cache Size:2
2022-02-28 11:27:07,711 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (100)
2022-02-28 11:27:07,714 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(MSISDN_PREFIX_LENGTH)
2022-02-28 11:27:07,716 [main] INFO com.dbrepository.DBConnectionPool - Active Size:2 Cache Size:1
2022-02-28 11:27:07,831 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (2)
2022-02-28 11:27:07,834 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(MAX_MSISDN_LENGTH)
2022-02-28 11:27:07,836 [main] INFO com.dbrepository.DBConnectionPool - Active Size:3 Cache Size:0
2022-02-28 11:27:08,393 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (15)
2022-02-28 11:27:08,395 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(USER_PRODUCT_MULTIPLE_WALLET)
2022-02-28 11:27:08,398 [main] INFO com.dbrepository.DBConnectionPool - Active Size:4 Cache Size:2
2022-02-28 11:27:08,511 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (false)
2022-02-28 11:27:08,514 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(EXTERNAL_DATE_FORMAT)
2022-02-28 11:27:08,516 [main] INFO com.dbrepository.DBConnectionPool - Active Size:5 Cache Size:1
2022-02-28 11:27:08,625 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (yyyy/MM/dd)
2022-02-28 11:27:08,628 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DATE_FORMAT_CAL_JAVA)
2022-02-28 11:27:08,631 [main] INFO com.dbrepository.DBConnectionPool - Active Size:6 Cache Size:0
2022-02-28 11:27:09,019 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (yyyy/MM/dd)
2022-02-28 11:27:09,022 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(OTH_COM_CHNL)
2022-02-28 11:27:09,024 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-02-28 11:27:09,135 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (false)
2022-02-28 11:27:09,184 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DW_ALLOWED_GATEWAYS)
2022-02-28 11:27:09,185 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-02-28 11:27:09,469 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching System Preference: 
2022-02-28 11:27:09,471 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
	at org.postgresql.jdbc.PgResultSet.checkResultSet(PgResultSet.java:2772)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:1894)
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getString(NewProxyResultSet.java:4865)
	at com.dbrepository.PostGreSQLRepository.getSystemPreference(PostGreSQLRepository.java:238)
	at com.utils._masterVO.loadPreferences(_masterVO.java:350)
	at com.utils._masterVO.loadVO(_masterVO.java:76)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:09,473 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-02-28 11:27:09,474 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DEFAULT_LANGUAGE)
2022-02-28 11:27:09,476 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-02-28 11:27:09,606 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (en)
2022-02-28 11:27:09,608 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DEFAULT_COUNTRY)
2022-02-28 11:27:09,609 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:27:09,732 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (US)
2022-02-28 11:27:09,733 [main] INFO com.utils._masterVO - Display Locale set to: en_US
2022-02-28 11:27:30,806 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.ConnectException: Connection timed out: connect
2022-02-28 11:27:30,835 [main] INFO com.utils._masterVO - Loading Messages Resource properties File from: .//src//test//resources//sysconfig//English.properties
2022-02-28 11:27:30,877 [main] INFO com.utils._masterVO - Messages Resource Properties file Loaded successfully
2022-02-28 11:27:30,878 [main] INFO com.utils._masterVO - Loading Messages Resource properties File from: .//src//test//resources//sysconfig//C2S_English.properties
2022-02-28 11:27:30,888 [main] INFO com.utils._masterVO - Messages Resource Properties file Loaded successfully
2022-02-28 11:27:30,889 [main] INFO com.dbrepository.PostGreSQLRepository - Entered getGatewayDetails('EXTGW','USSD','SMSC')
2022-02-28 11:27:30,890 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:27:30,893 [main] INFO com.dbrepository.PostGreSQLRepository - getGatewayDetails() :: select query: SELECT rmg.gateway_code, mg.gateway_type, rmg.login_id, rmg.PASSWORD, rmg.content_type, rmg.service_port, mg.req_password_plain FROM message_gateway mg, req_message_gateway rmg WHERE mg.gateway_code = rmg.gateway_code AND rmg.gateway_code IN ('EXTGW','USSD','SMSC')
2022-02-28 11:27:31,119 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: [[Ljava.lang.Object;@5505ae1a[{{EXTGW,EXTGW,pretups,0971500a350af5c3d1c0b12221a0558c,XML,190,Y},{USSD,USSD,pretups,0971500a350af5c3d1c0b12221a0558c,XML,190,Y},{SMSC,SMSC,pretups,0971500a350af5c3d1c0b12221a0558c,PLAIN,190,Y}}]
2022-02-28 11:27:31,120 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting getGatewayDetails()
2022-02-28 11:27:31,124 [main] INFO com.utils._APIUtil - Entered getAPIServices(EXTGW)
2022-02-28 11:27:31,131 [main] INFO com.utils._APIUtil - Exiting getAPIServices([])
2022-02-28 11:27:31,133 [main] INFO com.utils._masterVO - Loading Gateway Cases Master file from: .//src//test//resources//config//GatewayMaster.xlsx
2022-02-28 11:27:31,247 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,253 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,255 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,266 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,298 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,330 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,331 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,346 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,347 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,348 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,349 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,352 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,353 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,354 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,354 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,384 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,385 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,389 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,389 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,390 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,391 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,391 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,392 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,392 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,392 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,393 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,393 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,394 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,394 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,394 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,395 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,395 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,395 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,396 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,397 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,397 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,398 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,398 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,398 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,399 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,399 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,400 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,400 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,400 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:27:31,712 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-02-28 11:27:31,714 [main] INFO com.utils._APIUtil - Entered getAPIServices(USSD)
2022-02-28 11:27:31,717 [main] INFO com.utils._APIUtil - Exiting getAPIServices([])
2022-02-28 11:27:31,719 [main] INFO com.utils._masterVO - Loading Gateway Cases Master file from: .//src//test//resources//config//GatewayMaster.xlsx
2022-02-28 11:27:31,919 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-02-28 11:27:31,922 [main] INFO com.utils._masterVO - Loading Gateway Cases Master file from: .//src//test//resources//config//GatewayMaster.xlsx
2022-02-28 11:27:32,055 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-02-28 11:27:53,092 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.ConnectException: Connection timed out: connect
2022-02-28 11:27:53,129 [main] INFO com.classes.BaseTest - Completed loading all methods from -masterVO class
2022-02-28 11:29:20,431 [main] INFO com.classes.UserAccessRevamp - Trying to get User with Access: R_O2C,E_PURCHASE
2022-02-28 11:29:20,479 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:20,480 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getUserNameByLogin()
2022-02-28 11:29:20,772 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching UserName: 
2022-02-28 11:29:20,773 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
	at org.postgresql.jdbc.PgResultSet.checkResultSet(PgResultSet.java:2772)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:1894)
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getString(NewProxyResultSet.java:4865)
	at com.dbrepository.PostGreSQLRepository.getUserNameByLogin(PostGreSQLRepository.java:2239)
	at com.classes.UserAccessRevamp.getUserWithAccessRevamp(UserAccessRevamp.java:106)
	at restassuredapi.test.FOCInitiateAndApproval.TestDataFeed(FOCInitiateAndApproval.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
	at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
	at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:29:20,776 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: USER_NAME (null)
2022-02-28 11:29:20,778 [main] INFO com.classes.UserAccessRevamp - UserAccess Returns: PARENT_NAME(Super Admin) | CATEGORY_NAME(Super Channel Admin) | USER_NAME(null) | LOGIN_ID(AUT_60804) | PASSWORD(Com@2468)
2022-02-28 11:29:21,485 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-02-28 11:29:21,485 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_01_Test_success()
2022-02-28 11:29:21,486 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:29:21,487 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:29:22,123 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:29:22,996 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyODA3LjQ0My40MDAiLCJyZXFHYXRld2F5VHlwZSI6IlJFU1QiLCJpc3MiOiJwcmV0dXBzLm9hdXRoIiwic2VydmljZVBvcnQiOiIxOTAiLCJtc2lzZG4iOiI3MjMyODQ2NjEzMjg0MjEiLCJleHAiOjMwMDAwMCwicmVxR2F0ZXdheUNvZGUiOiJSRVNUIiwiaWF0IjoxNjQ2MDQ3Njg3LCJqdGkiOiJkY2E1MGJkMi1lYjFkLTQ4MTUtOTkwMC0yNTk3ZDZlMjMyYzYifQ.jrTltxeG1Wa53V6gchveJ-FDFuHmGph3Pr2qQiMbTAg","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyODA3LjQ0My40MDAiLCJyZXFHYXRld2F5VHlwZSI6IlJFU1QiLCJpc3MiOiJwcmV0dXBzLm9hdXRoIiwic2VydmljZVBvcnQiOiIxOTAiLCJtc2lzZG4iOiI3MjMyODQ2NjEzMjg0MjEiLCJleHAiOjYwMDAwMCwicmVxR2F0ZXdheUNvZGUiOiJSRVNUIiwiaWF0IjoxNjQ2MDQ3Njg3LCJqdGkiOiI0NDYwZjQyNC1mZTZiLTQwNmMtYWQ5YS0yZTQwNzY5M2RlYmIifQ.D6EdQvJc9C3DrkjmyvBP_Dac6-wqtXkZsNGxb90fTK4","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:29:23,079 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:29:23,649 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:29:23,650 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:23,651 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:23,652 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:23,843 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:23,845 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:23,847 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:23,850 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:23,853 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:24,047 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:24,050 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:24,074 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:29:25,073 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1128.100001.","errorMap":null,"transactionId":"OT220228.1128.100001"}]}
2022-02-28 11:29:25,078 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:29:25,081 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:25,277 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:29:25,279 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:25,281 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:25,282 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:25,481 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:25,482 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:25,484 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:25,486 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:25,487 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:25,685 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:25,687 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:25,695 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:29:26,092 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241230","errorMsg":"dd/MM/yyyy is empty or invalid."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:29:26,106 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [400] but found [200]
2022-02-28 11:29:47,132 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.ConnectException: Connection timed out: connect
2022-02-28 11:29:47,970 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-02-28 11:29:47,970 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_01_Test_success()
2022-02-28 11:29:47,971 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:29:47,971 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:29:47,975 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:29:48,289 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyODMyLjc5NC4yMzM0IiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0NzcxMiwianRpIjoiYWViNDVlMTUtNGUyOS00OWYwLTk2MTAtNTNiNmY4MzAyYzY4In0.Xnt1yk2xROmJdrWVgCjDXqfFGLgCGKbZa6dRMZlqNdk","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyODMyLjc5NC4yMzM0IiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0NzcxMiwianRpIjoiNTQ3NzdhNGYtMTc4MS00ODNjLWE3ZjUtMGQ3MGU5YTVkYWYxIn0.-pJ1a9H3xj090WkWL2NO0mwayyf17wXGKk-nSRVW-Iw","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:29:48,293 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:29:48,951 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:29:48,952 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:48,955 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:48,957 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:49,088 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:49,089 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:49,090 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:49,091 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:49,094 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:49,340 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:49,343 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:49,348 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:29:49,853 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1128.100002.","errorMap":null,"transactionId":"OT220228.1128.100002"}]}
2022-02-28 11:29:49,857 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:29:49,859 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:50,057 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:29:50,059 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:50,060 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:50,062 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:50,262 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:50,263 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:50,264 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:29:50,266 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:29:50,267 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:29:50,468 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:29:50,471 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:29:50,477 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:29:50,980 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241230","errorMsg":"dd/MM/yyyy is empty or invalid."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:29:50,988 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [400] but found [200]
2022-02-28 11:30:12,028 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.ConnectException: Connection timed out: connect
2022-02-28 11:30:12,766 [main] INFO com.classes.UserAccessRevamp - Trying to get User with Access: R_O2C,E_PURCHASE
2022-02-28 11:30:12,794 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:12,795 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getUserNameByLogin()
2022-02-28 11:30:13,098 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching UserName: 
2022-02-28 11:30:13,099 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
	at org.postgresql.jdbc.PgResultSet.checkResultSet(PgResultSet.java:2772)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:1894)
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getString(NewProxyResultSet.java:4865)
	at com.dbrepository.PostGreSQLRepository.getUserNameByLogin(PostGreSQLRepository.java:2239)
	at com.classes.UserAccessRevamp.getUserWithAccessRevamp(UserAccessRevamp.java:106)
	at restassuredapi.test.FOCInitiateAndApproval.TestDataFeed(FOCInitiateAndApproval.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
	at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
	at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:30:13,104 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: USER_NAME (null)
2022-02-28 11:30:13,107 [main] INFO com.classes.UserAccessRevamp - UserAccess Returns: PARENT_NAME(Super Admin) | CATEGORY_NAME(Super Channel Admin) | USER_NAME(null) | LOGIN_ID(AUT_60804) | PASSWORD(Com@2468)
2022-02-28 11:30:13,693 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_02_Test_InvalidTxnID -------------------
2022-02-28 11:30:13,693 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_02_Test_InvalidTxnID()
2022-02-28 11:30:13,694 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:30:13,694 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:30:13,698 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:30:14,004 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyODU4LjUxNC4zNjIyIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0NzczOCwianRpIjoiMTA1ZmQ5Y2MtOTYxZS00ZDNjLTk3YTctYmRlYWIwZDJlYjNiIn0._j49P_Dq1Dbz5HDv9IFsnGAzrNZSu8Cnb-JqTaIUM6Q","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyODU4LjUxNC4zNjIyIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0NzczOCwianRpIjoiMGFkMzkyNTEtMjkwMC00MTEzLWJmOTgtNmU5Y2U5MTAyMzE1In0.Vaztk67Z7KJEVxxFuTZdUMrH0vAlLbJf2fwJg-k2KZ0","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:30:14,009 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:30:14,516 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:30:14,516 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:14,517 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:14,518 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:14,638 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:14,639 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:14,640 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:14,641 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:14,642 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:14,766 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:14,768 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:14,772 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:30:15,237 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1128.100003.","errorMap":null,"transactionId":"OT220228.1128.100003"}]}
2022-02-28 11:30:15,242 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:30:15,244 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:15,358 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:30:15,359 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:15,361 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:15,362 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:15,490 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:15,492 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:15,494 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:15,495 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:15,496 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:15,606 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:15,608 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:15,612 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:30:15,969 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241230","errorMsg":"dd/MM/yyyy is empty or invalid."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:30:15,974 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>400<br><b>Found: </b>400</pre>
2022-02-28 11:30:16,473 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for A_02_Test_InvalidTxnID -----------------
2022-02-28 11:30:16,539 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_02_Test_InvalidTxnID -------------------
2022-02-28 11:30:16,539 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_02_Test_InvalidTxnID()
2022-02-28 11:30:16,542 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:30:16,542 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:30:16,545 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:30:16,857 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTAxLjM2MS40OTcwIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc0MSwianRpIjoiYTY4MzhhMzktYjNhMi00YzgwLTllN2YtOGJhYzIwMjcwYzhiIn0.qW3_elMtrSXG_6S-qH-NkjuC2g2iYEPFrInziJd3GwU","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTAxLjM2MS40OTcwIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc0MSwianRpIjoiYWVlZjYzMjMtZTlkMC00ZjQ4LTg2ODMtNTE4NGIzMmJkZTZkIn0.So5-wuw6w1PeP5yS03OqQGVv7e0JTeqVSFFvXv8knPA","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:30:16,861 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:30:17,390 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:30:17,391 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:17,392 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:17,394 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:17,508 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:17,509 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:17,511 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:17,513 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:17,514 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:17,626 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:17,628 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:17,633 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:30:18,092 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1129.100001.","errorMap":null,"transactionId":"OT220228.1129.100001"}]}
2022-02-28 11:30:18,096 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:30:18,098 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:18,223 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:30:18,225 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:18,226 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:18,228 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:18,335 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:18,337 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:18,339 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:18,340 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:18,342 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:18,456 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:18,458 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:18,462 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:30:18,834 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241230","errorMsg":"dd/MM/yyyy is empty or invalid."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:30:18,839 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>400<br><b>Found: </b>400</pre>
2022-02-28 11:30:19,324 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for A_02_Test_InvalidTxnID -----------------
2022-02-28 11:30:19,395 [main] INFO com.classes.UserAccessRevamp - Trying to get User with Access: R_O2C,E_PURCHASE
2022-02-28 11:30:19,429 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:19,431 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getUserNameByLogin()
2022-02-28 11:30:19,664 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching UserName: 
2022-02-28 11:30:19,665 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
	at org.postgresql.jdbc.PgResultSet.checkResultSet(PgResultSet.java:2772)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:1894)
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getString(NewProxyResultSet.java:4865)
	at com.dbrepository.PostGreSQLRepository.getUserNameByLogin(PostGreSQLRepository.java:2239)
	at com.classes.UserAccessRevamp.getUserWithAccessRevamp(UserAccessRevamp.java:106)
	at restassuredapi.test.FOCInitiateAndApproval.TestDataFeed(FOCInitiateAndApproval.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
	at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
	at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:30:19,666 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: USER_NAME (null)
2022-02-28 11:30:19,669 [main] INFO com.classes.UserAccessRevamp - UserAccess Returns: PARENT_NAME(Super Admin) | CATEGORY_NAME(Super Channel Admin) | USER_NAME(null) | LOGIN_ID(AUT_60804) | PASSWORD(Com@2468)
2022-02-28 11:30:20,256 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_03_Test_InvalidCurrentStatus -------------------
2022-02-28 11:30:20,256 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_03_Test_InvalidCurrentStatus()
2022-02-28 11:30:20,257 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:30:20,257 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:30:20,260 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:30:20,558 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTA1LjA4Ny44NjE3IiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc0NSwianRpIjoiNGFkOTFjOTctM2YyYi00MmUwLWEyN2UtOTFjNWNiNzVlYjA5In0.sCTh4xaUpRs8GbyboHm8eoFaTkcZB4btQ7G4zO242Kg","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTA1LjA4Ny44NjE3IiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc0NSwianRpIjoiNGFiZDc3OGMtODM0MS00ZWFiLWE2ODYtZGRkMzRmMmM0YzE4In0.d66pbVwkcijNuDYuqSg73yTny9rdk9p4XuYAF21Zt8o","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:30:20,562 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:30:21,106 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:30:21,106 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:21,107 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:21,108 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:21,345 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:21,348 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:21,352 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:21,354 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:21,356 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:21,545 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:21,547 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:21,552 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:30:22,050 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1129.100002.","errorMap":null,"transactionId":"OT220228.1129.100002"}]}
2022-02-28 11:30:22,057 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:30:22,059 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:22,248 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:30:22,249 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:22,250 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:22,252 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:22,450 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:22,453 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:22,457 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:22,460 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:22,462 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:22,644 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:22,648 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:22,655 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:30:23,071 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241230","errorMsg":"dd/MM/yyyy is empty or invalid."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:30:23,076 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>400<br><b>Found: </b>400</pre>
2022-02-28 11:30:23,613 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for A_03_Test_InvalidCurrentStatus -----------------
2022-02-28 11:30:23,702 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_03_Test_InvalidCurrentStatus -------------------
2022-02-28 11:30:23,702 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_03_Test_InvalidCurrentStatus()
2022-02-28 11:30:23,705 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:30:23,705 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:30:23,710 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:30:24,158 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTA4Ljc1NS41NjAzIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc0OCwianRpIjoiY2Y0NDAzMDEtMzU4YS00OTIxLWJkOGUtMTQ1NTYxYmEzNmUxIn0.ZkPxvDPhPR2WFoGgVQXXfKMqyLjRfKHov1g9uVMjtRY","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTA4Ljc1NS41NjAzIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc0OCwianRpIjoiMzk0MzY4ODQtOWNiNS00OGU4LTg3YjUtYmEwODk1MDc4MGJhIn0.MKdud1d8T8nwSdzGIEAj8WuDhQfhxC7Bi_W3_R5VP6o","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:30:24,162 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:30:24,715 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:30:24,717 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:24,718 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:24,719 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:24,959 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:24,960 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:24,961 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:24,962 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:24,963 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:25,076 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:25,078 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:25,084 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:30:25,564 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1129.100003.","errorMap":null,"transactionId":"OT220228.1129.100003"}]}
2022-02-28 11:30:25,571 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:30:25,573 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:25,762 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:30:25,765 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:25,767 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:25,771 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:25,964 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:25,969 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:25,972 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:25,974 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:25,976 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:26,165 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:26,169 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:26,176 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:30:26,603 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241230","errorMsg":"dd/MM/yyyy is empty or invalid."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:30:26,607 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>400<br><b>Found: </b>400</pre>
2022-02-28 11:30:27,144 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for A_03_Test_InvalidCurrentStatus -----------------
2022-02-28 11:30:27,213 [main] INFO com.classes.UserAccessRevamp - Trying to get User with Access: R_O2C,E_PURCHASE
2022-02-28 11:30:27,244 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:27,245 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getUserNameByLogin()
2022-02-28 11:30:27,591 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching UserName: 
2022-02-28 11:30:27,592 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
	at org.postgresql.jdbc.PgResultSet.checkResultSet(PgResultSet.java:2772)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:1894)
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getString(NewProxyResultSet.java:4865)
	at com.dbrepository.PostGreSQLRepository.getUserNameByLogin(PostGreSQLRepository.java:2239)
	at com.classes.UserAccessRevamp.getUserWithAccessRevamp(UserAccessRevamp.java:106)
	at restassuredapi.test.FOCInitiateAndApproval.TestDataFeed(FOCInitiateAndApproval.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
	at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
	at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-02-28 11:30:27,596 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: USER_NAME (null)
2022-02-28 11:30:27,598 [main] INFO com.classes.UserAccessRevamp - UserAccess Returns: PARENT_NAME(Super Admin) | CATEGORY_NAME(Super Channel Admin) | USER_NAME(null) | LOGIN_ID(AUT_60804) | PASSWORD(Com@2468)
2022-02-28 11:30:28,099 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_04_Test_InvalidReciverMsisdn -------------------
2022-02-28 11:30:28,099 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_04_Test_InvalidReciverMsisdn()
2022-02-28 11:30:28,100 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:30:28,100 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:30:28,104 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:30:28,477 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTEzLjE1MS41NTQwIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc1MywianRpIjoiZWRjNTgwOTktODBjNy00NzU4LTgwOTUtOGQ4ZDUxNmRkMDUyIn0.dnJ50W3NlXwLHEQ15RoebUWRzDHdr9MScC1sFJTJp-I","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTEzLjE1MS41NTQwIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc1MywianRpIjoiNzIxNzc4ZjEtODNkYi00MDY1LTg3NjMtZWU1NDc3OTA2OWNlIn0.TrmFq3t1pDprAbWVOMBTZTPpvY0T66HJtt50Z5mzoi4","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:30:28,484 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:30:29,039 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:30:29,041 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:29,042 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:29,043 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:29,154 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:29,157 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:29,159 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:29,161 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:29,162 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:29,274 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:29,278 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:29,285 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:30:29,781 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1129.100004.","errorMap":null,"transactionId":"OT220228.1129.100004"}]}
2022-02-28 11:30:29,789 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:30:29,790 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:29,981 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:30:29,985 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:29,987 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:29,990 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:30,118 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:30,121 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:30,123 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:30,126 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:30,127 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:30,283 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:30,287 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:30,292 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:30:30,625 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241193","errorMsg":"Invalid receiver's msisdn 12345."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:30:30,634 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>400<br><b>Found: </b>400</pre>
2022-02-28 11:30:31,175 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for A_04_Test_InvalidReciverMsisdn -----------------
2022-02-28 11:30:31,248 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For A_04_Test_InvalidReciverMsisdn -------------------
2022-02-28 11:30:31,248 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered A_04_Test_InvalidReciverMsisdn()
2022-02-28 11:30:31,249 [main] INFO restassuredapi.test.FOCInitiateAndApproval - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-02-28 11:30:31,249 [main] INFO restassuredapi.test.FOCInitiateAndApproval - Entered Test_OAuthenticationTest()
2022-02-28 11:30:31,252 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-02-28 11:30:31,588 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTE2LjMxOC42NzUwIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjozMDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc1NiwianRpIjoiNDgwNDZiM2ItNTdlMy00OGJjLWJjY2YtZDNjM2I5NjIwYzc5In0.jugND2AB_6cpzkQK1UttK8LfwQ3FFsM7itqfSuF6mlY","refreshToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsb2dpbklkIjoiQVVUXzYwODA0IiwicmVxR2F0ZXdheUxvZ2luSWQiOiJwcmV0dXBzIiwidG9rZW5JZCI6IjIwMjIwMjI4MTEyOTE2LjMxOC42NzUwIiwicmVxR2F0ZXdheVR5cGUiOiJSRVNUIiwiaXNzIjoicHJldHVwcy5vYXV0aCIsInNlcnZpY2VQb3J0IjoiMTkwIiwibXNpc2RuIjoiNzIzMjg0NjYxMzI4NDIxIiwiZXhwIjo2MDAwMDAsInJlcUdhdGV3YXlDb2RlIjoiUkVTVCIsImlhdCI6MTY0NjA0Nzc1NiwianRpIjoiNTUyNWRmOTItNzdmNi00OTVkLWFhY2QtZGNjMTA3YzJkYTA2In0.CzvSqaZPpnA0Zeq7GoF-U7GVTrPc6_CNJCDwHxhPipU","status":200,"messageCode":"9020","message":"Success","errorMap":null}
2022-02-28 11:30:31,594 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>200<br><b>Found: </b>200</pre>
2022-02-28 11:30:32,167 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for Test_OAuthenticationTest -----------------
2022-02-28 11:30:32,168 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:32,169 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:32,170 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:32,391 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:32,394 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:32,397 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:32,399 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:32,401 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:32,591 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:32,594 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:32,601 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapiv1/foc/focInitiate
2022-02-28 11:30:34,502 [main] INFO restassuredapi.api.focInitiateApi.FOCInititaeAPI - {"service":"FOCRESP","referenceId":null,"status":"200","messageCode":"200","message":"1 out of 1 record(s) processed successfully","errorMap":{"masterErrorList":null,"rowErrorMsgLists":null},"successList":[{"status":200,"messageCode":"241129","message":"Transfer request has been successfully initiated with transaction ID OT220228.1129.100005.","errorMap":null,"transactionId":"OT220228.1129.100005"}]}
2022-02-28 11:30:34,508 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(SYSTEM_DATE_FORMAT)
2022-02-28 11:30:34,510 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:34,624 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yy)
2022-02-28 11:30:34,627 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:34,627 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:34,628 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:34,802 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:34,804 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:34,807 [main] INFO com.dbrepository.PostGreSQLRepository - Entered checkForLangCode(English)
2022-02-28 11:30:34,810 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-02-28 11:30:34,812 [main] INFO com.dbrepository.PostGreSQLRepository - checkForLangCode() :: select query: SELECT LANGUAGE_CODE from LOCALE_MASTER where NAME = ?
2022-02-28 11:30:35,001 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: LanguageCode (0)
2022-02-28 11:30:35,005 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting checkForLangCode()
2022-02-28 11:30:35,013 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/o2c/focApproval
2022-02-28 11:30:35,352 [main] INFO restassuredapi.api.focapproval.FOCApprovalAPI - {"service":"focApproval","referenceId":null,"status":"400","messageCode":null,"message":"All records contain error.","errorMap":{"masterErrorList":null,"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":null,"rowErrorMsgList":[{"rowErrorMsgLists":[{"rowValue":null,"rowName":null,"masterErrorList":[{"errorCode":"241193","errorMsg":"Invalid receiver's msisdn 12345."}],"rowErrorMsgList":null}]}]}]},"successList":[]}
2022-02-28 11:30:35,360 [main] INFO com.utils.Assertion - <pre><b>Expected: </b>400<br><b>Found: </b>400</pre>
2022-02-28 11:30:35,917 [main] INFO restassuredapi.test.FOCInitiateAndApproval -  -----------------  Test Case Execution completed for A_04_Test_InvalidReciverMsisdn -----------------
